content/pages/homepage.html

```diff
@@ -90,13 +90,17 @@ <h1 class="ui huge header">
             header="Work privately or publicly",
             color="violet") %}
           <p>
-            Easily share within your team or with the whole world.
-            Manage permissions yourself or automatically with GitHub.
+            Share content with just your team, or with the whole world.
+            Sync permissions from GitHub, or use custom permissions in our dashboard.
```

Comment:
We should avoid mentioning the dashboard here probably, as permissions extend to docs too.

```suggestion
            Sync permissions from GitHub, or manage custom permissions for your projects.
```